composer.json

```diff
@@ -19,31 +19,34 @@
         "ext-xml": "*",
         "composer-runtime-api": "^2.1",
         "composer/semver": "^3.2",
-        "doctrine/doctrine-bundle": "^2.4",
-        "doctrine/doctrine-migrations-bundle": "^3.2",
-        "doctrine/migrations": "^3.3",
-        "doctrine/orm": "^2.10",
+        "doctrine/doctrine-bundle": "*",
+        "doctrine/doctrine-migrations-bundle": "*",
+        "doctrine/migrations": "*",
+        "doctrine/orm": "*",
         "monolog/monolog": "^2.3",
         "ondrejmirtes/better-reflection": "^4.3.81",
         "phpbench/phpbench": "^1.2",
         "phpstan/phpstan": "^1.2",
+        "spryker-sdk/aop-sdk": "dev-master",
+        "spryker-sdk/async-api": "dev-master",
+        "spryker-sdk/sync-api": "dev-master",
         "spryker-sdk/composer-replace": "dev-master",
         "spryker-sdk/sdk-contracts": "dev-master",
-        "spryker-sdk/spryk": "^0.3.2",
+        "spryker-sdk/spryk": "dev-master as 0.3.0",
```

Comment:
We need to tag this first. I'm on it.

Reply:
I'm not sure that it's working with last spryk. did we have integration with last spryk?